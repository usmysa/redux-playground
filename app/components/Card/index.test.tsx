import { renderWithProviders } from "@/__tests__";
import { Card } from ".";

describe("Card", () => {
  it("renders correctly", () => {
    const { container } = renderWithProviders(
      <Card>
        <div>Card</div>
      </Card>,
    );

    expect(container).toMatchSnapshot();
  });
});
