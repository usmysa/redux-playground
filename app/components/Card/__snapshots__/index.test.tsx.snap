// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Card > renders correctly 1`] = `
<div>
  <div
    class="styles_wrapper__1dl4v454"
    style="--styleVars_padding__1dl4v451: var(--spacing-md__xzbfpk3); --styleVars_radius__1dl4v452: var(--spacing-md__xzbfpk3);"
  >
    <div>
      Card
    </div>
  </div>
</div>
`;
